### DISCUSSION TITLE
Busco equipo para el hackaton

### DISCUSSION THREAD

**Discussion by AlexLezcano** [Launch-X-Latam/MisionFrontEnd#563] [Team Posts]
Estaba poniéndome al día con las tareas y recién me uno a la búsqueda de equipo.
Si hay alguno que necesite un integrante agradecería me incluyeran. 
Saludos
Alex Lezcano

**Comment by LuisColli1** [comment]
Hola me interesa unirme contigo, mandame un correo por favor lx17197@innovaccion.mx

**Comment by AbnerHub** [comment]
Hola aún no tengo equipo ¿formamos uno? mi correo es lx02522@innovaccion.mx

**Comment by BrandonAzure** [comment]
Hola yo tampoco tengo equipo ¿puedo ser parte de un equipo con ustedes
?

**Comment by AbnerHub** [comment]
Hola aún puedo entrar lx02522@innovaccion.mx

**Comment by Juanggtz** [comment]
Hola, ¿puedo entrar a su equipo?, mi correo es: lx15882@innovaccion.mx

**Comment by Cposadas05** [comment]
Buena noche a todos, por cuestiones de tiempo y trabajo me ha costado ponerme al día.

Pero alguien me podría aceptar en su equipo?

Enviado desde mi Samsung Mobile de Claro
Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by gfrodriguez** [comment]
¿Cuándo es la hackaton?, estoy perdido con ese tema.

**Comment by CarlNauj** [comment]
Buenos días:

                Mi nombre es Juan Carreño; estamos en la misma, por asuntos de trabajos me cuesta de ponerme al día, pero si quieres formar un grupo, estoy dispuesto a participar.

                Atento a tu respuesta.

              Saluda atentamente a usted.

JUAN CARLOS CARREÑO POBLETE.

De: Cposadas05 ***@***.***>
Enviado el: lunes, 14 de noviembre de 2022 21:54
CC: Subscribed ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-backend] Busco equipo para el hackaton (#100)

Buena noche a todos, por cuestiones de tiempo y trabajo me ha costado ponerme al día.

Pero alguien me podría aceptar en su equipo?

Enviado desde mi Samsung Mobile de Claro
Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
